sale_order_engine_data/reports/sale_order_template.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="report_saleorder_document_engine_data" inherit_id="sale.report_saleorder_document">
+        <xpath expr="//div[@name='expiration_date']" position="after">
+            <div t-if="doc.engine_ref and doc.engine_hours" class="col-auto col-3 mw-100 mb-2" name="engine_data">
+                <span style="display: block">
+                    <strong>Engine number:</strong>
+                    <p class="m-0" style="display: inline-block" t-field="doc.engine_ref"/>
+                </span>
+                <span style="display: block">
+                    <strong>Engine hours:</strong>
+                    <p class="m-0" style="display: inline-block" t-field="doc.engine_hours"/>
+                </span>
+            </div>
+        </xpath>
+    </template>
+
+    <template id="report_saleorder_engine_data">
+        <t t-call="web.html_container">
+            <t t-foreach="docs" t-as="doc">
+                <t
+                    t-call="sale_order_engine_data.report_saleorder_document_engine_data"
+                    t-lang="doc.partner_id.lang"
+                />
+            </t>
+        </t>
+    </template>
```

Comment:
We only want to improve current sales quotation/sale order PDF report, not creating a new one, so we should remove this wrap template, that only is needed when we need to split a report in two different ones